apps/rtm/controllers/boilerplate.js

```diff
@@ -10,10 +10,6 @@ module.exports = class BoilerPlate extends Controller{
 
   getValues(req, res, next) {
     this.options.clearSession = false;
-    res.locals.backLink = '/';
-    if (this.getReports(req).length) {
-      res.locals.backLink = '/confirmation';
-    }
     super.getValues(req, res, next);
   }
 };
```

Comment:
Not part of this PR but this is a funny name for a controller. Its clearing session on get, does it do anything else, is it just a start controller?